policy-library/policies/constraints/Kptfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Google LLC
```

Comment:
As per previous comment, not sure if this is required.

Reply:
removed  too